src/ansys/fluent/core/launcher/process_launch_string.py

```diff
@@ -99,12 +114,31 @@ def _build_fluent_launch_args_string(**kwargs) -> str:
         launch_args_string += " -gpu"
     elif isinstance(gpu, list):
         launch_args_string += f" -gpu={','.join(map(str, gpu))}"
-    ui_mode = UIMode(kwargs.get("ui_mode"))
-    if ui_mode and ui_mode.get_fluent_value()[0]:
-        launch_args_string += f" -{ui_mode.get_fluent_value()[0]}"
+    ui_mode_input = kwargs.get("ui_mode")
+    ui_mode = _get_default_ui_mode(UIMode(ui_mode_input))
```

Comment:
Thank you @seanpearsonuk, had missed this part where we were over-writing the user's input as well.